### ISSUE TITLE
Auto-receive

### ISSUE THREAD

**Issue by yasharzb** [tdlib/td#769]
Is it possible to receive notifications? So i can just see the difference and if it's not 0 i use gc command.
like:

```
try{
    client.send(new TdApi.GetUpdate(fromMessageId, chatId));
    client.send(new TdApi.GetChatHistory(.....);
} catch (NoUpdateException e){
    print("No updates for chat: " + chatId);
}
```

**Comment by levlam** [comment]
The question doesn't make much sence for me, but it is very likely that [Getting Started](https://core.telegram.org/tdlib/getting-started) has answer to it.

**Comment by yasharzb** [comment]
Imagine i have channel_1 in my chat lists. I don't want to use `gc` command to check if there is new message or not i need something like `getNotification` of `getUnread` count. If their not null/0 then i use `gc`. I didn't find anything similar to these.

**Comment by levlam** [comment]
You should never use GetChats to check for new messages. Everything you need is written in the [Getting Started](https://core.telegram.org/tdlib/getting-started).

**Comment by yasharzb** [comment]
Tnx! I found my solution! 
I used GetChat(chatId) to receive the last message of the chat. But there is a problem with forwarded messages. It seems that the time difference between the forwarded messages is less than the thread update time. So if sb forward 4 messages to me; I'll see 4 similar messages (the content is equal to the last forwarded message) in my log, by using GetChat(chatId). Is there any optimal solution for it? According to the document:
> For optimal performance, the number of the returned messages is chosen by TDLib and can be smaller than the specified limit

So I tried not to use GetChatHistory method. Is there any other ways?
